test/LedSign.js

```diff
@@ -107,87 +107,50 @@ describe('LedSign', () => {
         .to.equal(VALID_SIGN_REQUEST.backgroundColor);
       expect(signResponse.textColor).to.equal(VALID_SIGN_REQUEST.textColor);
       expect(signResponse.borderColor).to.equal(VALID_SIGN_REQUEST.borderColor);
-      done();
     });
-    it('Should return statusCode 404 when the sign is down', done => {
+    it('Should return statusCode 404 when the sign is down', async () => {
       healthCheckMock.rejects(ERROR_MESSAGE);
-      chai
-        .request(app)
-        .post('/api/LedSign/healthCheck')
-        .then(function(res) {
-          expect(res).to.have.status(NOT_FOUND);
-          done();
-        })
-        .catch(err => {
-          throw err;
-        });
+      const response = await test.sendPostRequest(
+        '/api/LedSign/healthCheck', officer);
+      expect(response).to.have.status(NOT_FOUND);
     });
   });
 
   describe('/POST updateSignText', () => {
-    it('Should return statusCode 200 when the sign text is updated', done => {
+    it('Should return statusCode 200 when the sign' +
+      'text is updated', async () => {
       updateSignTextMock.resolves(SUCCESS_MESSAGE);
-      chai
-        .request(app)
-        .post('/api/LedSign/updateSignText')
-        .send(VALID_SIGN_REQUEST)
-        .then(function(res) {
-          expect(res).to.have.status(OK);
-          done();
-        })
-        .catch(err => {
-          throw err;
-        });
+      const response = await test.sendPostRequest(
+        '/api/LedSign/updateSignText', VALID_SIGN_REQUEST);
+      expect(response).to.have.status(OK);
     });
-    it('Should return statusCode 400 when the sign is down', done => {
+    it('Should return statusCode 400 when the sign is down', async () => {
```

Comment:
also another dumb question but why do we replace done w/ async

Reply:
async because the SceApiTester functions are async

Reply:
🤡 ok that makes sense